help_docs/lightbug_http.txt

```diff
@@ -0,0 +1,32 @@
+{'scope': {'type': 'http',
```

Comment:
Wasn't sure what this file is for, is this an example? Should we reference it in the README? Or is it just here for future use